ui/components/app/srp-input/srp-input.js

```diff
@@ -132,11 +131,10 @@ export default function SrpInput({ onChange, srpText }) {
           {srpText}
         </Text>
       </label>
-      <ActionableMessage
+      <BannerAlert
         className="import-srp__paste-tip"
-        iconFillColor="var(--color-info-default)"
-        message={t('srpPasteTip')}
-        useIcon
+        severity="info"
```

Comment:
Could we use the `Severity` enum from `ui/helpers/constants/design-system.ts` for the severities?
```
import { Severity } from '../../../helpers/constants/design-system';
```

```suggestion
        severity={Severity.Info}
```

Reply:
I think that makes sense!